src/PatientChecking/PatientChecking/Views/Dashboard/Home.cshtml

```diff
@@ -2,4 +2,67 @@
     ViewData["Title"] = "Dashboard";
     ViewData["Page"] = "Dashboard";
 }
-<p>This is dashboard</p>
+@section css {
+    <link href="@Url.Content("~/css/Dashboard.css")" rel="stylesheet" type="text/css" />
+}
+<script>
+    (function () {
+        $('#loading-div-background').show();
+        
+        $.ajax({
+            type: 'GET',
+            url: 'https://localhost:5001/Dashboard/getDashBoard',
+            success: function (data) {
+                $(".total-patient").text(data.numOfPatients);
+                $(".patient-inmonth").text(data.numOfPatientsInMonth + " new paitents/this month");
+                $(".total-appointment").text(data.numOfAppointments);
+                $(".appointment-inmonth").text(data.numOfAppointmentsInMonth + " new appointments/this month");
+                $(".appointment-intoday").text(data.numOfAppointmentsInToday + " new appointments/today");
+                $('#loading-div-background').hide();
+                $(".grid-wide").show();
+            }
+        })
+    })();
+</script>
+<div id="loading-div-background" style="position: fixed; top: 50%; left: 60%;">
+    <div id="loading-div" class="ui-corner-all">
+        <div id="loadingbar"><i class="fa fa-spinner fa-spin fa-3x fa-fw"></i></div>
+        <h2 style="color:gray;font-weight:normal;">Please wait....</h2>
```

Comment:
see commend above

Reply:
I customized Loading tag, so we can use to other places